built_in_tasks/manualcontrolmultitasks.py

```diff
@@ -252,6 +252,19 @@ class TrackingTask(ManualControlMixin, ScreenTargetTracking):
 
 class SequenceTask(ManualControlMixin, SequenceCapture):
     '''
-    Track moving target task
+    Sequence task
+    '''
+    pass
+
+class HandConstrainedSaccadeTask(ManualControlMixin, HandConstrainedEyeCapture):
+    '''
+    Saccade task while holding different targets by hand
     '''
     pass
+
+class SaccadeTask(ManualControlMixin, ScreenTargetCapture_Saccade):
+    '''
+    Saccade task
```

Comment:
Can you add more description? The major difference i think is that the color of the targets changes when the cursor enters them?